routes/home/index.tsx

```diff
@@ -79,10 +79,12 @@ export const handler: Handlers = {
       return ctx.render(data);
     } catch (error) {
       console.error("Error in handler:", error);
-      return ctx.render({
-        error: `Error: Internal server error`,
-        btcPrice: 0,
-        recommendedFee: 6,
+      return new Response("", {
```

Comment:
this should use ResponseUtil internalError - should fall through to the 302 redirect to 404